pipeline/metadata/satellite.py

```diff
@@ -158,10 +198,22 @@ def _read_satellite_tags(filename: str, line: str) -> Iterator[Row]:
   yield tags
 
 
+def _is_vantage_point_tag(row: Row) -> bool:
+  """Determine if a given tag should be applied to vantage point or received ips.
+
+  TODO: remove this function and instead do the partitioning
```

Comment:
Yeah, they should not be merged in the first place. You're merging to then separate. And the merged shuffling is less performant

Reply:
Agreed these should definitely not be merged. Unmerging them is complicated and unrelated to the main point of this PR though. I'm currently working in it in a [separate branch](https://github.com/censoredplanet/censoredplanet-analysis/compare/triple-cogroup...partition-tags).